composer.json

```diff
@@ -51,15 +51,22 @@
         "coding-standards-check/twigcs": [
             "twigcs web/profiles/custom/os2loop/themes/os2loop_theme/templates"
         ],
+        "coding-standards-check/prettier": [
+            "yarn prettier --check web/profiles/custom/os2loop/themes/os2loop_theme/*/*.scss"
```

Comment:
It might be better to use `… 'web/profiles/custom/os2loop/themes/os2loop_theme/**/*.scss'` to catch `scss` files in subfolders as well (cf. https://prettier.io/docs/en/cli.html#file-patterns).